src/endpoints.js

```diff
@@ -0,0 +1,126 @@
+  */
+  classes: 'https://www.dnd5eapi.co/api/classes/',
+  /* RETURNS:
+    {
+      "count": 12,
+      "results": [
+        {
+          "index": "barbarian",
+          "name": "Barbarian",
+          "url": "/api/classes/barbarian"
+        },
+        {
+          "index": "bard",
+          "name": "Bard",
+          "url": "/api/classes/bard"
+        }, {}, {}...
+      ]
+    }
+  */
+  spell: 'https://www.dnd5eapi.co/api/spells/${index}/',
+  /* ${index} = acid-arrow RETURNS:
+    {
+      "index": "acid-arrow",
+      "name": "Acid Arrow",
+      "desc": [
+        "A shimmering green arrow streaks toward a target within range and bursts in a spray of acid. Make a ranged spell attack against the target. On a hit, the target takes 4d4 acid damage immediately and 2d4 acid damage at the end of its next turn. On a miss, the arrow splashes the target with acid for half as much of the initial damage and no damage at the end of its next turn."
+      ],
+      "higher_level": [
+        "When you cast this spell using a spell slot of 3rd level or higher, the damage (both initial and later) increases by 1d4 for each slot level above 2nd."
+      ],
+      "range": "90 feet",
+      "components": [
+        "V",
+        "S",
+        "M"
+      ],
+      "material": "Powdered rhubarb leaf and an adder's stomach.",
+      "ritual": false,
+      "duration": "Instantaneous",
+      "concentration": false,
+      "casting_time": "1 action",
+      "level": 2,
+      "attack_type": "ranged",
+      "damage": {
+        "damage_type": {
+          "index": "acid",
+          "name": "Acid",
+          "url": "/api/damage-types/acid"
+        },
+        "damage_at_slot_level": {
+          "2": "4d4",
+          "3": "5d4",
+          "4": "6d4",
+          "5": "7d4",
+          "6": "8d4",
+          "7": "9d4",
+          "8": "10d4",
+          "9": "11d4"
+        }
+      },
+      "school": {
+        "index": "evocation",
+        "name": "Evocation",
+        "url": "/api/magic-schools/evocation"
+      },
+      "classes": [
+        {
+          "index": "wizard",
+          "name": "Wizard",
+          "url": "/api/classes/wizard"
+        }
+      ],
+      "subclasses": [
+        {
+          "index": "lore",
+          "name": "Lore",
+          "url": "/api/subclasses/lore"
+        },
+        {
+          "index": "land",
+          "name": "Land",
+          "url": "/api/subclasses/land"
+        }
+      ],
+      "url": "/api/spells/acid-arrow"
+    }
+  */
+  charSpells: 'https://www.dnd5eapi.co/api/classes/${charClass}/spells',
+  /* ${charClass} = warlock RETURNS:
+    {
+      "count": 64,
+      "results": [
+        {
+          "index": "chill-touch",
+          "name": "Chill Touch",
+          "url": "/api/spells/chill-touch"
+        },
+        {
+          "index": "eldritch-blast",
+          "name": "Eldritch Blast",
+          "url": "/api/spells/eldritch-blast"
+        }, {}, {} ...
+      ]
+    }
+  */
+}
```

Comment:
Oh, this file isn't my updated version.  I will need to push that up with the next PR.

Reply:
This is super helpful though. I love how you put comments with the format of the data you get back.